src/components/DataTable2/DataTable2.jsx

```diff
@@ -351,6 +351,26 @@ function DataTableCardPill2({isLoading, rowHeight}) {
 		}
 	}, [watchParams]);
 
+	// Get credentials id from localstorage
+	function getCredentialsIdFromLS () {
+		let ls;
+		if (localStorage) {
+			try {
+				ls = JSON.parse(localStorage.getItem('Credentials'));
+			} catch (e) {
+				/*Ignore*/
+			}
+		}
+		return ls ? ls?.credentials : [];
+	}
+
+	// Getting a username if the value matches the credentials id
+	function getUsernameOrValue(valueArray) {
```

Comment:
I think this is not sustainable (together with getCredentialsIdFromLS). I think we have to come up with other sustainable way (any injection??) of how to get translations for credentials. It would worth to ask on review/planning